tf2onnx/function/range.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""
+tf2onnx.tf2onnx - range op conversion
+"""
+import numpy as np
+from onnx import helper
+from onnx.onnx_pb import TensorProto
+from tf2onnx import utils
+
+# pylint: disable=useless-return,broad-except,logging-not-lazy,unused-argument,missing-docstring
+def make_range_const(ctx,
+                     start_node,
+                     limit_node,
+                     delta_node,
+                     output,
+                     name,
+                     dtype):
+    """make Range subgraph if all inputs are const."""
+    # T range = Range(T start, T limit, T delta)
+    # V v_final_and_scan_outputs = Loop(int64 M, B cond, V v_initial)
+    base_name = utils.make_name(name)
+    start = start_node.get_tensor()
+    limit = limit_node.get_tensor()
+    delta = delta_node.get_tensor()
+    val = np.arange(start, limit, delta, dtype=start.dtype)
+    const_range = ctx.make_const(base_name, val)
+    return ctx.make_node("Identity", [const_range.output[0]], dtypes=[dtype], outputs=[output])
+
+def make_range_subgraph(ctx, start, limit, delta, output, name, dtype):
+    """make Range subgraph."""
```

Comment:
make_range_non_const

Reply:
accept start/limit/delta as node instead of str (output name), to keep the interface consistent.